sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/aspnethostedservice/Directory.Build.targets

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <!-- Use decentralized package references when building outside https://github.com/Azure/azure-sdk-for-net -->
+  <ItemGroup Condition="'$(IsSamplesProject)' != 'true'">
```

Comment:
We'll want to be sure to do this after the import, so that we're covered in case the definition of `IsSamplesProject` moves around to the repo targets instead of the props where it lives today.   Unlikely that it would happen, but there's no downside to flipping the order here.